include/conf/FaasmConfig.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
I couldn't bring myself to add another Faasm-specific parameter to the Faabric `SystemConfig`, so this is the beginnings of splitting out the Faasm-specific config from there.